lpa-submissions-e2e-tests/cypress/support/common/verifySectionName.js

```diff
@@ -1,11 +1,8 @@
 /// <reference types = "Cypress"/>
-import CommonPageObjects from '../PageObjects/CommonPageObjects';
-
-const commonPageObjects = new CommonPageObjects();
+import { sectionName } from '../PageObjects/common-page-objects';
 
 module.exports = (section) => {
-  commonPageObjects
-    .sectionName()
+  sectionName()
```

Comment:
As per previous comments, looks like an unnecessary abstraction. Could remove a number of page objects this way perhaps?

Reply:
Pushed back.